Library/Formula/truncate.rb

```diff
@@ -0,0 +1,21 @@
+class Truncate < Formula
+  desc "This program truncates a file to a given size"
+  homepage "http://www.vanheusden.com/truncate"
+  url "http://www.vanheusden.com/truncate/truncate-0.8.tgz"
+  sha256 "165914d643351324d69096427d3ce5d3ab13aed3f7c990a71ba3bd0c6f278f00"
+
+  def install
+    inreplace "truncate.c" do |s|
+      s.gsub! '#include "error.h"', ""
+      s.gsub! "off64_t", "off_t"
+      s.gsub! "stat64", "stat"
+      s.gsub! "ftruncate64", "ftruncate"
+    end
```

Comment:
What are these modifications for? Is there a bug to fix? We should use a patch since those don’t seem to be strictly related to Homebrew.